src/kafka-util/src/client.rs

```diff
@@ -92,19 +95,19 @@ impl<C> BrokerRewritingClientContext<C> {
     /// Connections to the specified `broker` will be rewritten to connect to
     /// `rewrite_host` and `rewrite_port` instead. If `rewrite_port` is omitted,
     /// only the host is rewritten.
-    pub fn add_broker_rewrite(
-        &mut self,
-        broker: &str,
-        rewrite_host: &str,
-        rewrite_port: Option<u16>,
-    ) {
+    ///
+    /// Note that this will not override an existing rewrite.
```

Comment:
This is a bit surprising, given that the default behavior of `insert` methods in Rust is to overwrite. Can we add a `if_not_exists` suffix to the method to make it very clear?